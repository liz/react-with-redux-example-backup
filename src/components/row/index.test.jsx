import React from 'react';
import { mount } from 'enzyme';

import { Row } from './';
import mediaQueries from '../../media-queries';

describe('Row', () => {
    it('should match the snapshot', () => {
        const wrapper = mount(<Row />);

        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders Container with expected props', () => {
        const wrapper = mount(<Row />);

        expect(wrapper.find('Row')).toHaveLength(1);
        expect(wrapper.find('Row').props()).toEqual({
            breakPoint: mediaQueries.min.medium,
            height: null,
        });
    });
});