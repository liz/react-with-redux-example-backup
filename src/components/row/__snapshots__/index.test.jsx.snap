// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row should match the snapshot 1`] = `"<div class=\\"row__RowContainer-g7667b-0 fhhpBZ\\"></div>"`;
